tests/post_training/test_quantize_conformance.py

```diff
@@ -69,40 +70,126 @@ def fixture_extra_columns(pytestconfig):
     return pytestconfig.getoption("extra_columns")
 
 
-@pytest.fixture(scope="session", name="reference_data")
-def fixture_reference_data():
-    path_reference = Path(__file__).parent / "reference_data.yaml"
+@pytest.fixture(scope="session", name="ptq_reference_data")
+def fixture_ptq_reference_data():
+    path_reference = Path(__file__).parent / "data" / "ptq_reference_data.yaml"
     with path_reference.open() as f:
         data = yaml.safe_load(f)
     return data
 
 
-@pytest.fixture(scope="session", name="result_data")
-def fixture_report_data(output_dir, run_benchmark_app, extra_columns):
+@pytest.fixture(scope="session", name="wc_reference_data")
+def fixture_wc_reference_data():
+    path_reference = Path(__file__).parent / "data" / "wc_reference_data.yaml"
+    with path_reference.open() as f:
+        data = yaml.safe_load(f)
+        fp32_test_cases = defaultdict(dict)
+        for test_case_name in data:
+            model_name = test_case_name.split("_backend_")[0]
+            fp32_test_cases[f"{model_name}_backend_FP32"]["metric_value"] = 1
+        data.update(fp32_test_cases)
+    return data
+
+
+@pytest.fixture(scope="session", name="ptq_result_data")
+def fixture_ptq_report_data(output_dir, run_benchmark_app):
     data: Dict[str, RunInfo] = {}
 
     yield data
 
     if data:
         test_results = OrderedDict(sorted(data.items()))
-        df = pd.DataFrame(v for v in test_results.values())
-
+        df = pd.DataFrame(v.get_result_dict() for v in test_results.values())
         if not run_benchmark_app:
             df = df.drop(columns=["FPS"])
-        if not extra_columns:
```

Comment:
why do you remove it?

Reply:
because it won't be filled here anyway.
https://github.com/openvinotoolkit/nncf/pull/2425/files#diff-e1768fd4dbf70fbbfb13bcd0a6283a19e817877024ac2864740ada6f7502b7fcR231-R232